XCast/XCast.h

```diff
@@ -118,6 +120,8 @@ class XCast : public PluginHost::IPlugin, public PluginHost::JSONRPC, public RtN
     bool deleteFromDynamicAppCache(JsonArray applications);
     bool deleteFromDynamicAppCache(vector<string>& appsToDelete);
     void updateDynamicAppCache(JsonArray applications);
+    void getSystemPlugin();
+    int updateSystemFriendlyName();
```

Comment:
Add deprecated label for the get/setFriendly Name APis in XCast. For instance https://github.com/rdkcentral/rdkservices/blob/main/SystemServices/System.json#L351